src/components/Modal/index.web.js

```diff
@@ -40,6 +43,15 @@ function Modal(props) {
             onModalHide={hideModal}
             onModalShow={showModal}
             avoidKeyboard={false}
+            coverScreen={false}
+            customBackdrop={(onBackdropPress) => (
+                <Pressable
+                    onPress={onBackdropPress}
+                    style={styles.modalBackdropWeb}
+                    accessibilityLabel="backdrop"
+                    accessibilityRole={CONST.ACCESSIBILITY_ROLE.ADJUSTABLE}
+                />
+            )}
```

Comment:
I don't think they are needed, since this is a fix only for mobile safari, we do not open keyboard on desktop, and the app works correctly there anyway